### PR TITLE
support for foreign key

### PR DESCRIPTION
**Pull request by sockeqwe**
Discussed in #18

### CODE REVIEW

**Comment by nhattan123** [comment]
Hi! Sockeqwe !you can help me! Write code times zone perfecting pleae

**Comment by sockeqwe** [comment]
?

**Comment by minamak91** [comment]
@sockeqwe why you added the support for only the build version below 16 ?

File: dao/src/main/java/com/hannesdorfmann/sqlbrite/dao/DaoManager.java [hunk 1]
```diff
@@ -130,9 +138,26 @@ public static Builder with(Context context) {
...
+      this.foreignKeyConstraints = foreignKeyConstraints;
+    }
+
+    @Override public void onOpen(SQLiteDatabase db) {
+      super.onOpen(db);
+      if (Build.VERSION.SDK_INT < 16) {
+        if (foreignKeyConstraints) {
+          db.execSQL("PRAGMA foreign_keys=ON;");
+        }
+      }
```
Comment (minamak91) [inline dao/src/main/java/com/hannesdorfmann/sqlbrite/dao/DaoManager.java:156]: @sockeqwe  Why here only below 16 ?

**Pull request merged**